src/views/UIOptionsPopup/style.ts

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components'
+
+export const UIOptionsDialog = styled.div`
+  top: 40px;
+  right: 16px;
+  width: 38em;
+  position: absolute;
+  background-color: rgba(0, 0, 0, 0.9);
+  background-opacity: 0.5;
```

Comment:
Shouldn't need this property if you're using rgba.

Reply:
Again, thanks!

Reply:
Done